api/request.go

```diff
@@ -28,21 +48,46 @@ type RequestedResource struct {
 	// Warning: We do not expect storage accounts to have snapshots. If your
 	// storage account has any, please contact System Admin, as due to caching
 	// you might end up with incorrect data.
-	Vds string `json:"vds" binding:"required" example:"https://account.blob.core.windows.net/container/blob"`
+	// The vds URL can either be provided as a single string or a list of strings.
+	Vds stringOrSlice `json:"vds" binding:"required" example:"https://account.blob.core.windows.net/container/blob" or ["vds_url_1", "vds_url_2"]`
 
 	// A valid sas-token with read access to the container specified in Vds
-	Sas string `json:"sas,omitempty" example:"sp=r&st=2022-09-12T09:44:17Z&se=2022-09-12T17:44:17Z&spr=https&sv=2021-06-08&sr=c&sig=..."`
+	// The sas-token can either be provided as a single string or a list of strings.
+	Sas stringOrSlice `json:"sas,omitempty" example:"sp=r&st=2022-09-12T09:44:17Z&se=2022-09-12T17:44:17Z&spr=https&sv=2021-06-08&sr=c&sig=..." or ["sas1", "sas2"]`
+
+	// When two VDS URL´s are provided the *binary_operator* defines how data
+	// from these sources are combined. Provided VDS A and VDS B and the
+	// **subtraction** operation request returns data from (A - B).
+	// Valid options are: **addition**, **subtraction**, **multiplication**, **division** and empty string.
+	BinaryOperator string `json:"binary_operator,omitempty" example:"subtraction"`
```

Comment:
Do we state anywhere the requirements for the vds-es?
Like that they should have exactly same axes and so on?

We also must think what to write in the docs regarding index coordinate system if we may drop support for this case in the future.
Now users can state "get me fence with traces in index coordinate system [0, 0]", but if coordinate system requirement would be relaxed, we would either make an exception for equal layout systems, or forbid it (which would create a breaking change as something stops working) or would do something like "first vds is a priority vds and all index coordinates are given in it".
So maybe it would be nice to put some kind of warning for future changes somewhere.

You also made a really good point today. We are missing origin comparison check. Now someone can provide to identical surveys taken in two different places and get the result. Only if they ask for data in UTM coordinates, would they notice that something is absolutely, definitely off.
So maybe we should mention too that origin is not taken into account.

(we could also benefit from some core_test.go slice/test/attributes tests with compatible/incompatible coordinate systems and various ways of asking for the data, but I suspect you will anyway make lots of them in your next PRs)